src/main.ts

```diff
@@ -63,6 +63,12 @@ export default class AdvancedURI extends Plugin {
             callback: () => this.handlers.handleCopyFileURI(true),
         });
 
+        this.addCommand({
```

Comment:
I would either add the markdown version for the command with options as well, or just the one with options.